src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -190,20 +200,31 @@ const SampleChatInfoScreen = () => {
     isAnonymousModalOpen,
     blockModalRef,
     handleShowArrow,
-    loadingChannelInfo
+    loadingChannelInfo,
+    goToEditGroup
   } = useChatInfoScreenHook();
   const [isLoadingMembers] = React.useState<boolean>(false);
   const {signedProfileId} = useUserAuthHook();
   const [profile] = (React.useContext(Context) as unknown as any).profile;
   const {params}: any = useRoute();
+  const isEditable = channelInfo?.rawJson?.channel?.channelType === 1;
   const ANONYMOUS_USER = 'AnonymousUser';
   const {anon_user_info_color_code, anon_user_info_emoji_code} =
     channelInfo?.rawJson?.channel || {};
+  const {anonProfileId} = useProfileHook();
+  const memberChat = channelInfo?.rawJson?.channel?.members?.find(
+    (item: any) => item.user_id !== anonProfileId
+  );
+  const betterSocialMember = channelInfo?.rawJson?.better_channel_member;
   const showImageProfile = (): React.ReactNode => {
     if (channelInfo?.channelType === CHANNEL_GROUP) {
       return (
         <ChannelImage>
-          <ChannelImage.Big type={GROUP_INFO} image={channelInfo?.channelPicture} />
+          <ChannelImage.Big
+            style={channelImageStyles.containerImageGroupINfo}
+            type={GROUP_INFO}
+            image={channelInfo?.channelPicture}
+          />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `goToEditGroup`, `isEditable`, `anonProfileId`, `memberChat`, `betterSocialMember` and their usage in the code enhances the functionality of the component by providing more context and control over the chat group editing process. However, ensure that these new hooks and variables are properly initialized and handled in their respective contexts.



<!-- This is an auto-generated comment by OSS CodeRabbit -->